src/NServiceBus.Core/Extensibility/ExtendableOptions.cs

```diff
@@ -5,6 +5,6 @@ namespace NServiceBus.Extensibility
     /// </summary>
     public abstract class ExtendableOptions
```

Comment:
It's the "hook" where all capabilities/transports will add their extension methods?